schema/site_metadata.json

```diff
@@ -50,6 +58,42 @@
             "type": "number"
         }
       }
+    },
+    "area": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "m2": {
+            "description": "Area of the site in square meters",
+            "type": "number",
+            "examples": [1234, 256, 998765]
+        },
+        "sqf": {
+            "description": "Area of the site in square feet",
```

Comment:
Redundancy is usually a bad idea if there's really only one thing, because it only means that somebody can get it wrong. Why two fields?  I'd go with one field with a units -- otherwise you're implying that the m2 could be different than sqf...

Reply:
We need to consume this information in separate formats. The logic to ensure the information comes from a single field will be in the data entry tool (spreadsheet based, locked field+formula)